src/DashboardUI/src/pages/application/create/ApplicationCreatePage.tsx

```diff
@@ -0,0 +1,398 @@
+import { useNavigate } from 'react-router-dom';
+import MapProvider from '../../../contexts/MapProvider';
+import { ApplicationNavbar } from '../components/ApplicationNavbar';
+import { useConservationApplicationContext } from '../../../contexts/ConservationApplicationProvider';
+import Form from 'react-bootstrap/esm/Form';
+import { NotImplementedPlaceholder } from '../../../components/NotImplementedAlert';
+import FloatingLabel from 'react-bootstrap/esm/FloatingLabel';
+import Button from 'react-bootstrap/esm/Button';
+import { states } from '../../../config/states';
+import {
+  CompensationRateUnitsLabelsPlural,
+  CompensationRateUnitsLabelsSingular,
+  CompensationRateUnitsOptions,
+} from '../../../data-contracts/CompensationRateUnits';
+import { formatNumber } from '../../../utilities/valueFormatters';
+
+export function ApplicationCreatePage() {
+  const { state } = useConservationApplicationContext();
+  const navigate = useNavigate();
+
+  const navigateToEstimationToolPage = () => {
+    navigate(`/application/${state.conservationApplication.waterRightNativeId}/estimation`);
+  };
+
+  return (
+    <MapProvider>
+      <div className="application-create-page d-flex flex-column flex-grow-1 h-100">
+        <ApplicationNavbar
+          navigateBack={navigateToEstimationToolPage}
+          backButtonText="Back to Estimator"
+          centerText="Water Conservation Estimation Tool"
+        />
+
+        <ApplicationCreatePageForm />
+      </div>
+    </MapProvider>
+  );
+}
+
+const emptyStringPlaceholder = '';
+function ApplicationCreatePageForm() {
+  const { state } = useConservationApplicationContext();
+
+  // assumes all polygons are not intersecting
+  const acreageSum = state.conservationApplication.selectedMapPolygons.reduce(
+    (sum, polygon) => sum + polygon.acreage,
+    0,
+  );
+  const etAcreFeet = state.conservationApplication.polygonEtData.reduce(
+    (sum, polygon) => sum + polygon.averageYearlyEtInAcreFeet,
+    0,
+  );
+
+  return (
+    <div className="flex-grow-1 overflow-y-auto p-4">
```

Comment:
I'd recommend using a container here to help with responsive layouts
```suggestion
    <div className="container">
```

Reply:
very smart idea